pages/blog/fetch-for-servers.mdx

```diff
@@ -0,0 +1,171 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'JavaScript runs everywhere, so should your servers - here is how',
+  tags: ['fetch', 'whatwg', 'node', 'graphql', 'graphql-yoga'],
+  authors: ['arda'],
+  date: '2022-07-21',
+  description: 'The new way of building servers in JavaScript',
+  image:
+    'https://user-images.githubusercontent.com/727224/104328926-56b0a700-54ba-11eb-9ad3-f6d2b971d55d.jpg',
+  thumbnail:
+    'https://user-images.githubusercontent.com/727224/104328777-29fc8f80-54ba-11eb-87d0-9982af1d8281.jpg'
+}
+
+export default withArticle(meta)
+
+At the beginning of the year, we’ve launched GraphQL Yoga 2.0 - a server framework for GraphQL APIs.
+
+While planning version 2.0 of Yoga, we were thinking about all the things that changed in the ecosystem and what developers using JavaScript expect from their server frameworks now and in the future.
+
+One of the most powerful trend in the JS ecosystem was the proliferation of new environments and platforms that can run JS(Lambdas, Cloudflare Workers, Deno, Bun etc.). So we set up to build a single GraphQL server framework that could run on any of these platforms.
+
+What we’ve found in the process was fascinating and we believe would change how any JS HTTP frameworks are being built, without any relation to GraphQL
+
+While Node.js is the most popular environment, multiple platforms can run JavaScript with different APIs. Usually, they have their own way of creating servers, but we saw they share a web standard that most people usually use for communicating with the backends from the client side, not the other way around. That’s WHATWG Fetch API that uses `Request`, `Response`, `ReadableStream`, etc to manage this on the server side, too.
+
```

Comment:
I'm not sure what you meant here so I suggested some changes by having some assumptions:

```suggestion
While Node.js is the most popular environment, multiple platforms can run JavaScript, usually, having their own way of creating servers, and using different APIs.

On the other hand, the JavaScript client-side has mostly migrated over the years to a common set of standards (`fetch`), independently of the underlying platform.

This is where we realized that WHATWG Fetch API that uses `Request`, `Response`, and `ReadableStream` could also be used on the server side to provide a unified API to run JavaScript HTTP servers everywhere.

```